client/state/sites/features/reducer.js

```diff
@@ -46,6 +70,13 @@ export function featuresReducer( state = {}, { type, siteId, features, error } )
 				error: error,
 				isRequesting: false,
 			} );
+		case JETPACK_SITES_FEATURES_FETCH:
+			return { ...state, isRequestingJetpackSitesFeatures: true };
```

Comment:
I don't see a good reason to nest `isRequestingJetpackSitesFeatures` under `features` TBH. I actually find it confusing - that list is supposed to contain a list of features, keyed by site ID, and then we contain `isRequestingJetpackSitesFeatures` there as well. Sounds like a recipe for trouble because we could easily lose the extra key when working to update the site features.

I think it makes more sense to have `isRequestingJetpackSitesFeatures` in another sub-reducer tree. What do you think?

Reply:
Yes I agree, I moved it to a different sub tree.